README.md

```diff
@@ -39,14 +39,35 @@ How
 
 ### Deploy
 
+Revolver is packaged as an AWS Lambda function and is triggered by a Cloudwatch Event.
+
 This repository only contains the Revolver code and no deployment mechanisms. To prepare it for deploying you should use `npm run build` and `npm run bundle` commands to build the code and create a zip file.
 suitable for deployment in AWS Lambda.
 
+You are free to choose your own deployment mechanism. We use [CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html) to deploy Revolver.
+
 ### Configure
 
-Revolver configuration is done in YAML. First line in the config file must be `---` as per YAML specification.
+Revolver reads some of the low-level configuration from environment variables and the rest from a YAML file in S3.
+
+#### Environment variables
+
+|Variable|Description|Default|
+|-|-|-|
+|S3_BUCKET|S3 bucket where the config file is stored|-|
+|S3_KEY|S3 key of the config file|-|
+|DEBUG_LEVEL|Log level|info|
+|LOG_FORMAT|Log format|pretty|
```

Comment:
please format table.  markdownlint etc can do it for you.

Reply:
it passes markdownlint, can we pls do it in the next release